flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogTable.java

```diff
@@ -121,6 +126,40 @@ public void testCreateTable() throws TableNotExistException {
     Assert.assertEquals(Maps.newHashMap(), catalogTable.getOptions());
   }
 
+  @Test
+  public void testCreateTableWithPrimaryKey() throws Exception {
```

Comment:
is there an update primary key syntax in flink?

Reply:
We apache flink don't support altering primary key now, but we have [proposal](https://issues.apache.org/jira/browse/FLINK-21634?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel) to extend the flink's alter table. Maybe we will support it in the following releases.